dxcherry/src/info/index.ts

```diff
@@ -0,0 +1,15 @@
+import { window } from 'vscode';
+import log from './log';
+
+function logInfo (msg: string) {
```

Comment:
why error log implementation is not the same?

Reply:
I just didn't find any reason to unify it. 

We need 3 api endpoints for our custom errors: throw, show (using `window.showErrorMessage`) and log.
And 2 for work with other info: show (using `window.showInfoMessage`) and log.

If we try to use the same logging function for errors and info we'll get the following problems:
-  any code related to errors will be more uglier (we'll have to give up an idea about predefined errors using [sample](https://github.com/ksercs/dx-gitAutomation/pull/8/files#diff-b1833d22f7ecf86fb5686d26e7af63f3654960eb58708b15af476b152d2841c6R22))
- we'll have no ability to differ error and info logging in the code visually
now: 
```
    logInfo('Start pull requests creating');
    missingGithubTokenError.log();
```
will be: 
```
    logInfo('Start pull requests creating');
    logInfo('Missing Github token');
```

Reply:
I mean logError/showError and logInfo/showInfo and remove all `throw`.

Reply:
Moreover, logError and logInfo should add something like 'ERROR' and 'INFO' at the beginning of the log.